recipe/meta.yaml

```diff
@@ -1,78 +1,257 @@
-    - jupyter nbconvert tests/files/notebook1.ipynb --to html
-    # skipping test_post_processor and test_convert_full_qualified_name as they expect tests to be a module
-    - pytest -v -k "not (test_post_processor or test_convert_full_qualified_name)"
+
+outputs:
+  - name: nbconvert
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage("nbconvert-core", exact=True) }}
+        - {{ pin_subpackage("nbconvert-pandoc", exact=True) }}
+    test:
+      requires:
+        - pip
+      commands:
+        - pip check
+        - jupyter nbconvert --version
+        - jupyter dejavu --version
+        - jupyter nbconvert --help
+        - jupyter dejavu --help
+
+  - name: nbconvert-core
+    build:
+      script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+      entry_points:
+        - jupyter-nbconvert = nbconvert.nbconvertapp:main
+        - jupyter-dejavu = nbconvert.nbconvertapp:dejavu_main
+    requirements:
+      host:
+        - python
+        - pip
+        - hatchling >=1.5
+      run:
+        - python
+        - beautifulsoup4
+        - bleach !=5.0.0
+        - defusedxml
+        - importlib-metadata >=3.6  # [py<310]
+        - jinja2 >=3.0
+        - jupyter_core >=4.7
+        - jupyterlab_pygments
+        - markupsafe >=2.0
+        - mistune >=2.0.3,<4
+        - nbclient >=0.5.0
+        - nbformat >=5.7
+        - packaging
+        - pandocfilters >=1.4.1
+        - pygments >=2.4.1
+        - traitlets >=5.1
+        - tinycss2
+      run_constrained:
+        - pyqtwebengine >=5.15
+        - tornado >=6.1
+        # other packages carry the full `run` dependency
+        - pandoc >={{ min_pandoc }},<{{ max_pandoc }}
+        # avoid mixing nbconvert-core and pre-split nbconvert
+        # can't mix noarch:generic and noarch:python with exact=True
+        - nbconvert ={{ version }}=*_{{ build }}
+
+    test:
+      imports:
+        - nbconvert
+        - nbconvert.exporters
+        - nbconvert.filters
+        - nbconvert.postprocessors
+        - nbconvert.preprocessors
+        - nbconvert.resources
+        - nbconvert.utils
+        - nbconvert.writers
+      source_files:
+        - tests
+      requires:
+        - pip
+        - pytest >=7
+        - ipykernel
+        - ipywidgets >=7.5
+        - flaky
+      commands:
+        - pip check
+        - jupyter nbconvert --version
+        - jupyter dejavu --version
+        - jupyter nbconvert --help
+        - jupyter dejavu --help
+        - jupyter nbconvert tests/files/notebook1.ipynb --to html
+        - pytest -vv {{ pytest_args }}
+    about:
+      home: https://jupyter.org
+      license: BSD-3-Clause
+      license_file: LICENSE
+      description: nbconvert with extra packages for browser-based PDF generation
+      doc_url: https://nbconvert.readthedocs.org/
+      dev_url: https://github.com/jupyter/nbconvert
+
+  - name: nbconvert-qtpdf
```

Comment:
We've got pyqtwebengine - are you removing this because of the test errors, which aren't worth it to solve them?

Also, maybe comment out for clarity

Reply:
I commented below that `qtpdf` implicitly requires `xelatex` on `linux` and `PasteBoard` on `osx` otherwise a `Segmentation fault` happens. I can move the comment before the output name.

Reply:
Done